docs/app/data-sharing.md

```diff
@@ -0,0 +1,219 @@
+Share.
+
+:::
+
+## Limit Sharing by Collection
+
+You can limit which Collections a Role is allowed to share.
+
+1. Navigate to **Settings > Roles and Permissions**.
+2. Select the **[Role Name]** that will be responsible for creating the Share.
+3. Find the **[Collection]** you wish to set Filter(s) for and click the icon in the column under
+   <span mi icon>share</span> and a small menu will appear.
+4. Select <span mi>check</span> to enable and <span mi>block</span> to disable Shares on this collection.
+
+## Limit Sharing by Item
+
+Sometimes you may want a given Role to be able to share some Items but not others. Filters allow you to control exactly
+which Items a Role will be able to share, enabling you to do things like _only share reports that are marked as 'Done'_.
+
+1. Navigate to **Settings > Roles and Permissions**.
+2. Select the **[Role Name]** that will be responsible for creating the Share.
+3. Find the **[Collection]** you wish to set Filter(s) for and click the icon in the column under
+   <span mi icon>share</span> and a small menu will appear.
+4. Select <span mi icon>rule</span>**Use Custom** and a side menu will appear.
+5. Create your Filter(s) as needed.
+
+## Limit Sharing by Field
+
+As mentioned several times before, when you create a Share on some Item, you associate a Role _(and thus the Role's
+permissions)_ to the Item. This allows you to set read permissions for the exact fields _(including relational fields)_
+that get shared.
+
+1. Navigate to **Settings > Roles and Permissions**.
+2. Select the **[Role Name]** that is assigned to the Share.
+3. Find the **[Collection]** and click the icon in the column under the <span mi icon>visibility</span> icon and a small
+   menu will appear.
+4. Select <span mi icon>rule</span>**Use Custom** and a side menu will appear.
+5. Click **Field Permissions**.
+6. Click to check the Collection Fields you'd like to allow read permissions on.
+7. Select <span mi>check</span> in the side menu Header to confirm changes.
+
+## Extensibility Options
+
+Directus Core is completely open-source, modular and extensible. Extensions allow you to expand or modify any part of
+Directus to fit your needs. Here are some great resources to get started down that track.
+
+- [Extensions > Introduction](/extensions/introduction/)
+- [Extensions > Creating Extensions](/extensions/creating-extensions/)
+- [Contributing > Introduction](/contributing/introduction/)
+- [Contributing > Codebase Overview](/contributing/codebase-overview/)
+
+::: tip Sponsoring Accelerated Development
+
+Working on an enterprise project and looking to outsource or financially sponsor the development of a Shares extension?
+Contact [our team](https://directus.io/contact/).
+
+:::
+
+## More Help
+
+Looking for technical support for your non-enterprise project? Chat with thousands of engineers within our growing
+[Community on Discord](https://discord.com/invite/directus).
```

Comment:
Lets not include this in the page content. If we want to have a consistent footer for every page, it should be on the theme level, not the individual content of the pages.

Besides, for the context of Shares, extensions don't really come into play, so it doesn't have to be on this page regardless.

```suggestion
```